pyvista/plotting/plotting.py

```diff
@@ -324,7 +330,10 @@ def track_click_position(self, side="right", callback=None):
         def _click_callback(obj, event):
             self.store_click_position()
             if hasattr(callback, '__call__'):
-                try_callback(callback, self.click_position)
+                if viewport:
+                    try_callback(callback, self.click_position)
+                else:
+                    try_callback(callback, self.pick_click_position())
```

Comment:
My justification is that `click_position` is a property because it is a a static object whereas `pick_click_position` is a method because it is a command performing a task (the click position can have varying picks depending on the camera's status). I chose this out of consistency with the other picking methods

Reply:
Fair enough.